run.sh

```diff
@@ -25,4 +25,4 @@ cd data
 # We are using `exec` to start Sia Ant Farm in order to ensure that it will be
 # run as PID 1. We need that in order to have Sia Ant Farm receive OS signals
 # (e.g. SIGTERM) on container shutdown, so it can exit gracefully.
-exec sia-antfarm -config=../$CONFIG
\ No newline at end of file
+exec sia-antfarm-dev -config=../$CONFIG
```

Comment:
Q: I find it odd that the executable name changes here. I suppose that's because we're now using the `dev` image but I still find it surprising. Are you sure this is the right name? If so, then all is fine, I just wanted to make sure.

Reply:
Yes, using `dev` executable was an intended change.